doc/examples/transform/plot_register_align.py

```diff
@@ -0,0 +1,73 @@
+from skimage.data import camera
+from scipy import ndimage as ndi
+from skimage.transform import register_affine
+from matplotlib import pyplot as plt
+import numpy as np
+from skimage import measure
+
+rows=6
```

Comment:
PEP8 style recommends spaces on either side of the equals assignment operator - can you change this to match? Thanks!

https://www.python.org/dev/peps/pep-0008/ 
"Always surround these binary operators with a single space on either side: assignment (=), augmented assignment (+=, -= etc.), comparisons (==, <, >, !=, <>, <=, >=, in, not in, is, is not), Booleans (and, or, not)."

Reply:
(Might be worth taking another quick check over the rest of the code with autopep8)